README.md

```diff
@@ -16,9 +16,9 @@
 
 ## Why DuraCloud?
 
-In the digital era, ensuring that critically important documents remain safe and available is a continual challenge. Physical computing hardware that is used to create and store documents can fail or become obsolete very quickly, providing a clear need for decisive action to ensure that these documents remain available.
+In the digital era, ensuring that critically important documents remain safe and available is a continual challenge. Physical computing hardware that is used to create and store documents can fail or become obsolete very quickly.
```

Comment:
We may be missing a transition between this paragraph and the next. It may not be immediately clear that failing hardware results in a need for file storage and backup.

Reply:
Maybe the solution is simpler. Like, "Physical computing hardware that is used to create and store documents can fail or become obsolete very quickly, providing a need for tools to ensure that these documents remain available." This would only remove the words "clear" and "decisive action".